cmd/root.go

```diff
@@ -82,6 +85,8 @@ func init() {
 	f.BoolVarP(&RootArgs.force, "force", "f", false, "Force the copy automatically overwriting everything at destination.")
 	f.IntVarP(&RootArgs.batchSize, "batch-size", "b", 4096, "The number of documents to write at once.")
 	f.IntVarP(&RootArgs.maxRetries, "max-retries", "r", 9, "The number of maximum retries attempts. Increasing this number will also increase the exponential fallback timer.")
+	f.BoolVar(&RootArgs.simpleProgress, "simple-progress", false, "Simplify the progress so it can be viewed on simpler terminal session like, remote screen session.")
```

Comment:
Can we automate this?

Reply:
I don't believe so, because it's not clear in what type of terminal the current progress works. I can also imagine that some people just don't like it, and would like a simpler one which just outputs things on the line.

Also the remote session is still considered to be a tty which breaks progress bar apparently. So there is no definite way of detecting whether the progress would work or not.